jina/orchestrate/flow/base.py

```diff
@@ -2408,6 +2436,9 @@ def to_kubernetes_yaml(
                 )
                 if node == 'gateway'
                 else None,
+                k8s_deployments_metadata=self._get_k8s_deployments_metadata()
```

Comment:
revert

Reply:
@JoanFM Could you explain why this function is not necessary?